tests/test_orchestration/test_slurm_sharding.py

```
@@ -56,6 +56,37 @@ def _mode(path):
     return path.stat().st_mode & 0o777
 
 
+def _make_single_node_container_config():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_make_single_node_container_config`

(ANN202)